### DISCUSSION TITLE
Lost abreviation shortcuts

### DISCUSSION THREAD

**Discussion by sebw** [autokey/autokey#780] [General]
Hi,

I've been running autokey for years, and my config files are stored in my dotfiles managed by stow.

I just noticed that I somewhat "lost" all my abbreviation shortcuts.

If I inspect the json file I can still see the abbreviation so I reset the abbreviation for one shortcut, and I see the json file is now stored as `.blah.json`, while my existing config was stored as `blah.json`.

I could quickly do a quick rename of all json files in my dotfiles, but is there any specific change in behavior introduced recently?

Thanks!

**Comment by josephj11** [comment]
Welcome to the AutoKey community, sebw!

First guess is that at some point you upgraded to 0.96.0 and then downgraded to 0.95.10 or older. (or are accessing a shared config on the cloud with both versions of AutoKey on different machines, etc.)

0.96.0 renamed all the metadata files so that they are no longer hidden. This was incompatible with previous versions - which is likely what you are experiencing.

If that's the case, the recommended solution (other than just using 0.96.0) would be to create symlinks to all the .json files with the same name as they have now, but with a prefix of a period. That way, both versions can find them. Be sure not to do this for any you have "fixed" manually because two leading periods won't work either.

0.96.0 has been out for a while and is quite stable. It includes a lot of bug fixes and several new features. You should use it unless you have a compelling reason not to. And, if you do have such an issue, please report it so we can address it.

Hopefully, this will get you back into your "blahs" again. ;)  (Although I'm not sure if that's entirely a good thing.)

**Reply by sebw** [reply]
Thanks @josephj11 for confirming this change :) Despite my research attempts I couldn't find anything relevant.

I've been distro hoping and moved from Fedora to Void to Arch. 

Void didn't have autokey as a package so I installed from pip but I don't recall the version. Same fore Fedora before that. Now in Arch I installed from AUR and I'm in version 0.96.0.

I will just proceed with the symlinks, this is a great idea.

Cheers!

**Comment by josephj11** [comment]
If you're sticking with 0.96.0, you don't need the symlinks, but they shouldn't hurt anything.

See first item [here](https://github.com/autokey/autokey/releases/).

When a project changes their top version number (95 to 96 - since the developers aren't ready to declare that we're at 1.0), it's a good idea to look at the change logs, because that's when breaking changes happen. And, that's where the new features are all listed.

Also:

if you haven't visited our [wiki](https://github.com/autokey/autokey/wiki) recently, there's a lot of new content  and it's better organized.

Our [API](https://github.com/autokey/autokey/wiki/API-Examples) documentation had a major face lift as well.
